server/service/http/message/query/row.rs

```diff
@@ -29,9 +37,9 @@ pub fn encode_row(
         let variable_value = row.get(*position);
         let row_entry =
             encode_row_entry(variable_value, snapshot, type_manager, thing_manager, include_instance_types)?;
-        encoded_row.insert(variable, row_entry);
+        encoded_row.insert(variable.as_str(), row_entry);
     }
-    Ok(json!(encoded_row))
+    Ok(json!(EncodedRow { data: encoded_row, provenance: row.provenance().0.to_le_bytes() }))
```

Comment:
Now encoded as le_bytes()

Reply:
Sick - I think you might as well throw this into `Provenance::to_array` right?

Reply:
Hmm really what you want is a 64-bit array right? I'm actually not convinced the 8x8 bit array is much better than 1 u64... what if we call the network field `provenance_bit_array: u64` instead?

Reply:
We could, but that's pretty verbose. We could also encode it as a hex-string.

Reply:
I'm happy with calling it (only when we encode it into the network) as `provenance_bit_array` and not change the rest of the server :)